ONSdigital/response-operations-ui
# [TBL] Call party direct when resending verification email

# Motivation and Context

Removes the call to backstage for resending verification email.

# What has changed



Changes the `resend_verification_email` method in reporting_units_controllers.py to hit the party service directly.

# How to test?



Run the unit tests, make sure they pass.
Tail the logs for backstage, party service. Hit the `/reporting-units/resend_verification/<ru_ref>/<party_id>` page. Make sure backstage doesn't get the call, but party service does.
